docs/windowsClinic.md

```diff
@@ -0,0 +1,76 @@
+# Running full clinic deployment on Windows
+
+### Requirements
+
+Setup was tested on Windows 10 Home Edition.
+
+## 1. Install Docker Toolbox for Windows.
+
+*   Unless you have them already installed choose to install Virtual Box and Git for Windows.
+*   Run Docker Toolbox QuickStart Shell to create and setup dockem machine.
+
+## 2. Checkout IRYO WWM repo.
+
+Before checkout, while configuring git you should disable auto conversion to CRLF line endings.
+
+```
+git config --global core.autocrlf false
+```
+
+## 3. Add IRYO WWM dir to shared folders for docker machine VM.
+
+Run in powershell (as admin) while being in IRYO WWM dir:
+
+```
+docker-machine stop
```

Comment:
in my setup `docker-machine stop` failed to actually stop the VM running inside VirtualBox. docker-machine's state put the "machine" to stopped mode and lost an ability to manage VMs in virtualbox.

Reply:
This setup with virtualbox feels very fragile to me. I was only able to run the `vboxmanage ...` comand after manually opening the vbox file located in `~/.docker/machine/machines/default/default/default.vbox`. After that, the VM actually got listed in virtualbox's UI and was available to `vboxmanage`. I mean it's not something we can't work around, but does feel broken.

Reply:
For now I've changed this description to just set it up in Virtual Box GUI when using Windows Home. This seems to work when you do not touch anything after initial Docker Toolbox installation. I've also added Windows Pro setup.